.github/workflows/continuous-integration.yml

```diff
@@ -135,8 +131,8 @@ jobs:
       - name: "Create build directory"
         run: "mkdir -p .build/logs"
 
-      - name: "Collect code coverage with Xdebug and phpunit/phpunit"
-        run: "vendor/bin/phpunit --coverage-clover=.build/logs/clover.xml"
+      - name: "Collect code coverage with Xdebug and phpunit"
+        run: "vendor/bin/simple-phpunit --coverage-clover=.build/logs/clover.xml"
```

Comment:
I am not sure that this is necessary.

Reply:
I tried it with regular phpunit and got this:
```
line 1: vendor/bin/phpunit: No such file or directory
```
It is necessary